### PR TITLE
Add instructions for an included datadog_archives sink.

### PR DESCRIPTION
**Pull request by silversupreme**
### What does this PR do? What is the motivation?
We've decided to include a sample `datadog_archives` sink in the included configurations for Linux, Docker, Terraform, and EKS. This PR includes instructions for how to setup a customer's environment to properly use this config.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by jszwedko — COMMENTED** [review]
👍 Do we already have docs about how to rehydrate from these? Or are those forthcoming? If we have them already, we should link from here.

File: content/en/observability_pipelines/setup/datadog.md [hunk 1]
```diff
@@ -41,11 +41,85 @@ You can generate both of these in [Observability Pipelines][3].
...
+        ]
+      }
+    ]
+  }
+  ```
+
+* Edit the bucket name in the policy above to match your configuration.
+
+* Create an IAM user and attach the policy above to it, and create access credentials for it. Save these credentials as `AWS_ACCESS_KEY` and `AWS_SECRET_ACCESS_KEY`.
 {{% /tab %}}
```
Comment (maycmlee) [inline content/en/observability_pipelines/setup/datadog.md:81]:
```suggestion

[1]: https://aws.amazon.com/s3/pricing/

{{% /tab %}}
```

File: content/en/observability_pipelines/setup/datadog.md [hunk 2]
```diff
@@ -89,16 +165,125 @@ See [Best Practices for OPW Aggregator Architecture][1] for production-level req
...
```
Comment (maycmlee) [inline content/en/observability_pipelines/setup/datadog.md:204]:
```suggestion

[1]: https://aws.amazon.com/s3/pricing/

{{% /tab %}}
```

File: content/en/observability_pipelines/setup/datadog.md [hunk 3]
```diff
@@ -89,16 +165,125 @@ See [Best Practices for OPW Aggregator Architecture][1] for production-level req
...
+        ]
+      }
+    ]
+  }
+  ```
+
+* Edit the bucket name in the policy above to match your configuration.
+
+* Create an IAM user and attach the policy above to it, and create access credentials for it. Save these credentials as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
 {{% /tab %}}
```
Comment (maycmlee) [inline content/en/observability_pipelines/setup/datadog.md:242]:
```suggestion

[1]: https://aws.amazon.com/s3/pricing/

{{% /tab %}}
```

**Review by maycmlee — COMMENTED** [review]
Left some suggestions! There were also a couple of links missing so I added them in.

**Review by maycmlee — CHANGES_REQUESTED** [review]
Left some suggestions! Can we also:

- Rename the file from `archives.md` to `datadog_with_archiving.md`.
- Remove CloudFormation content, since it doesn't have any archiving info (I don't think)?

Jesse also mentioned adding a link to the Rehydrating doc: https://docs.datadoghq.com/logs/log_configuration/rehydrating/

File: content/en/observability_pipelines/setup/datadog_with_archiving.md [hunk 4]
```diff
@@ -0,0 +1,628 @@
...
+kind: Documentation
+further_reading:
+  - link: "/observability_pipelines/production_deployment_overview/"
+    tag: "Documentation"
+    text: "Production deployment design and principles for the Observability Pipelines Worker"
+  - link: "https://dtdg.co/d22op"
+    tag: "Learning Center"
+    text: "Safe and Secure Local Processing with Observability Pipelines"
+---
+
```
Comment (maycmlee) [inline content/en/observability_pipelines/setup/datadog_with_archiving.md:12]:
```suggestion

{{< site-region region="gov" >}}
<div class="alert alert-warning">Observability Pipelines is not available on the US1-FED Datadog site.</div>
{{< /site-region >}}

```

File: config/_default/menus/menus.en.yaml [hunk 5]
```diff
@@ -2939,6 +2939,10 @@ main:
     url: observability_pipelines/setup/datadog
     parent: observability_pipelines_setup
     weight: 101
+  - name: Datadog with Archiving
+    url: observability_pipelines/setup/datadog_with_archiving
+    parent: observability_pipelines_setup
+    weight: 101
```
Comment (maycmlee) [inline config/_default/menus/menus.en.yaml:2945]:
```suggestion
    weight: 102
```

File: content/en/observability_pipelines/setup/datadog_with_archiving.md [hunk 6]
```diff
@@ -0,0 +1,628 @@
...
+* A Pipeline ID.
+
+You can generate both of these in [Observability Pipelines][3].
+
+### Provider-specific requirements
+{{< tabs >}}
+{{% tab "Docker" %}}
+Ensure that your machine is configured to run Docker.
+
+The sample configuration you download later includes a sink for sending logs to Amazon S3 under a Datadog-rehydratable format. To use this configuration,  set up an IAM policy that allows the Workers to write to S3.
```
Comment (maycmlee) [inline content/en/observability_pipelines/setup/datadog_with_archiving.md:37]:
```suggestion
The sample configuration you download later includes a sink for sending logs to Amazon S3 under a Datadog-rehydratable format. To use this configuration, set up an IAM policy that allows the Workers to write to S3.
```

File: content/en/observability_pipelines/setup/datadog_with_archiving.md [hunk 7]
```diff
@@ -0,0 +1,628 @@
...
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+
+The Observability Pipelines Worker Docker image is published to Docker Hub [here][1].
+
+1. Download the [sample pipeline configuration file][2].
+
+2. Run the following command to start the Observability Pipelines Worker with Docker:
+    ```
```
Comment (maycmlee) [inline content/en/observability_pipelines/setup/datadog_with_archiving.md:284]: Want to see if I can fix the markdown...for some reason the rest of the content is in blue

```suggestion

    ```shell
```
```

**Review by maycmlee — APPROVED** [review]
@barieom did you want to review as well?

**Pull request closed without merging**